PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -15,6 +15,14 @@
     cut = cms.string("pt > 18 && tauID('decayModeFindingNewDMs') && (tauID('byLooseCombinedIsolationDeltaBetaCorr3Hits') || tauID('byVLooseIsolationMVArun2v1DBoldDMwLT2015') || tauID('byVLooseIsolationMVArun2v1DBnewDMwLT') || tauID('byVLooseIsolationMVArun2v1DBdR03oldDMwLT') || tauID('byVVLooseIsolationMVArun2v1DBoldDMwLT') || tauID('byVVLooseIsolationMVArun2v1DBoldDMwLT2017v2') || tauID('byVVLooseIsolationMVArun2v1DBnewDMwLT2017v2') || tauID('byVVLooseIsolationMVArun2v1DBdR03oldDMwLT2017v2') || tauID('byVVVLooseDeepTau2017v2VSjet'))")
```

Comment:
@mbluj are we sure this is correct? We are asking for the deepTau bit also in cases where it is not there, e.g. UL2017, and it will crash there, right? I can fix it with the nanoAOD PR, could you please confirm?

Reply:
DeepTauID **2017v2** is contained in MiniAOD of UL2017 as it is now, i.e. w/o planned reMiniAOD, so check for `tauID('byVVVLooseDeepTau2017v2VSjet')` is correct.
What concerns planned UL2018/2017 and UL2017 reMiniAOD it will contain  DeepTauID **2017v2p1** so this check is modified to `tauID('byVVVLooseDeepTau2017v2p1VSjet')` with corresponding era modifiers. 
Regardless, I think that matrix tests will fail in case of inconsistent configurations of Mini and NanoAOD.

Reply:
ok, sorry, I did not read carefully enough and I thought we were asking for 2p1